app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -6320,55 +6321,62 @@ function lookupNetworkTests({
   operation: (controller: NetworkController) => Promise<void>;
 }) {
   describe('if the network ID and network details requests resolve successfully', () => {
-    describe('if the current network is different from the network in state', () => {
-      it('updates the network in state to match', async () => {
-        await withController(
-          {
-            state: initialState,
-          },
-          async ({ controller }) => {
-            await setFakeProvider(controller, {
-              stubs: [
-                {
-                  request: { method: 'net_version' },
-                  response: { result: '12345' },
-                },
-              ],
-              stubLookupNetworkWhileSetting: true,
-            });
+    const validNetworkIds = [12345, '12345', toHex(12345)];
```

Comment:
Nit: Is it worth adding a test for an invalid network ID? Could do this in a new PR as well.

Reply:
Good thought, but we already covered this below (using the id `invalid`).